src/js/storyTelling-blockUi-timeBased.js

```diff
@@ -89,12 +89,14 @@ https://raw.githubusercontent.com/fluid-project/sjrk-story-telling/master/LICENS
         // for the source element of the mediaPlayer.
         // Alternatively, we could go with something simpler based on directly using the
         // src attribute of the video or audio element.
-        var source = mediaPlayer.find("source");
-        if (source.length < 1) {
-            source = $("<source></source>").appendTo(mediaPlayer);
+
+        if (mediaUrl) {
```

Comment:
This if block can be simplified to: `mediaPlayer.prop("controls", !!mediaUrl);`

Reply:
I've never seen that "operator" before! very neat. thanks for the tip :) I'll make that change once I return to working on this branch